projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/dag_validator.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+from collections import namedtuple
+from typing import Dict
+from typing import List
+
+from vdk.internal.core.errors import ErrorMessage
+from vdk.internal.core.errors import UserCodeError
+
+log = logging.getLogger(__name__)
+Error = namedtuple("Error", ["TYPE", "PERMISSION", "REQUIREMENT", "CONFLICT"])
+ERROR = Error(
+    TYPE="type", PERMISSION="permission", REQUIREMENT="requirement", CONFLICT="conflict"
+)
+allowed_job_keys = ["job_name", "team_name", "fail_meta_job_on_error", "depends_on"]
```

Comment:
use set instead of list
```suggestion
allowed_job_keys = {"job_name", "team_name", "fail_meta_job_on_error", "depends_on"}
```

Reply:
Done.